qiita_pet/handlers/study_handlers/description_handlers.py

```diff
@@ -582,6 +582,33 @@ def display_template(self, study, user, msg, msg_level, top_tab=None,
                     sub_tab=sub_tab,
                     prep_tab=prep_tab)
 
+    def delete_raw_data(self, study, user, callback):
+        """Delete the selected raw data
+
+        Parameters
+        ----------
+        study : Study
+            The current study object
+        user : User
+            The current user object
+        callback : function
+            The callback function to call with the results once the processing
+            is done
+        """
+        raw_data_id = self.get_argument('raw_data_id')
```

Comment:
`get_argument` always comes in as a string. It needs to be typecast to an int before use.